src/stylesheets/components/_buttons.scss

```diff
@@ -2,118 +2,57 @@
 
 button {
   cursor: pointer;
+  color: inherit;
   @include epsilon;
-}
-
-.btn {
-  @include btn;
-}
-
-/*--- Default Button Toggle ---*/
-
-.btn--toggle {
-  border: 0;
-  display: none;
-  position: relative;
+  @include typography--heavy;
   background-color: transparent;
-  padding-right: $base-spacing-unit-small;
-
-  &::after {
-    content: '';
-    display: inline-block;
-    position: absolute;
-    top: 50%;
-    right: 0;
-    vertical-align: middle;
-    border: $base-spacing-unit solid transparent;
-    border-bottom-width: 0;
-    border-top-color: $grey-1;
-    margin: -($base-spacing-unit / 2) 0 0 $base-spacing-unit-tiny;
-  }
-}
-
-.has-js .btn--toggle {
-  display: inline;
-}
-
-/*--- Button Toggle [within the header] ---*/
-
-header .btn--toggle {
-  top: $base-spacing-unit;
-  right: 0;
-  width: 57px;
-  height: 50px;
-  position: absolute;
-  background-color: $deep-purple;
-  padding: $base-spacing-unit $base-spacing-unit-small;
-  @media (max-width: $medium - 1) {
-    display: inline;
-  }
+  padding: 0;
+  border: 0;
 
-  &:after {
-    display: none;
-  }
-  &:hover, &:focus, &:active {
-    outline: none;
-
-    svg[data-icon="search"] g {
-      fill: $grey-3;
-    }
-    svg[data-icon="close"] g {
-      stroke: $grey-3;
-    }
+  &:hover, &:focus {
+    text-decoration: underline;
   }
 }
 
-.has-js header .btn--toggle {
-  @include media($medium) {
-    display: none;
-  }
+.btn {
+  @include btn;
 }
 
-/*--- Button Toggle within components ---*/
+/*--- Dropdown buttons ---*/
 
 .dropdown,
 .dropdown--block {
-  .btn--toggle {
+  > button {
+    display: none;
+    position: relative;
+    text-align: left;
     margin: 0;
   }
-}
-
-.dropdown.open,
-.dropdown--block.open {
-  .btn--toggle::after {
+  > button::after {
+    content: "";
+    display: inline-block;
+    vertical-align: middle;
+    border-top-width: $base-spacing-unit;
+    border-top-style: solid;
+    border-right: $base-spacing-unit solid transparent;
+    border-left: $base-spacing-unit solid transparent;
+    border-bottom-width: 0;
+    margin: -($base-spacing-unit / 2) 0 0 $base-spacing-unit;
+  }
+  &.open > button::after {
     transform: rotate(180deg);
   }
 }
 
-.dropdown--block .btn--toggle {
-  border: 0;
+.dropdown--block > button {
   width: 100%;
-  display: inline;
-  text-align: left;
+  background-color: transparent;
```

Comment:
Don't think this is necessary. Only a `button` without a class would be used that doesn't have a `background-color`?